src/trace/step-function-service.ts

```diff
@@ -3,23 +3,57 @@ import { SampleMode, TraceSource } from "./trace-context-service";
 import { SpanContextWrapper } from "./span-context-wrapper";
 import { Sha256 } from "@aws-crypto/sha256-js";
 
-export interface StepFunctionContext {
```

Comment:
since the context is also used as `tags` here [code line](https://github.com/DataDog/datadog-lambda-js/blob/8087c872b1806b27c2d6cface2f1f6b4b702815c/src/trace/listener.ts#L289) can we clarify why the change of the tags in the PR description?